docs/ccoctl.md

```diff
@@ -161,3 +161,98 @@ $ ccoctl gcp delete --name=<name> --project=<gcp-project-id>
 ```
 
 where `name` is the name prefix used to create cloud resources. `project` is the ID of the gcp project.
+
+## Alibaba Cloud
+
+This part is the guide for using  manual mode on alibaba cloud, for more use info about manual mode, please reference to [cco-mode-manual](https://github.com/openshift/cloud-credential-operator/blob/master/docs/mode-manual-creds.md).
+
+In alibaba cloud manual mode,  the CCO utility (`ccoctl`) binary will create Secret manifests for the OpenShift installer that will use a single long-lived RAM AK credentials for all OpenShift Container Platform cluster components. The target ram user who owns the AK would be attached various ram policies with the permission defined in each component's CredentialsRequest, and a root ram user who required `ram:CreatePolicy` and `ram:AttachPolicyToUser` pemission as least is needed for attaching the permission/policy to the target ram user.
+
+### Prerequisite
+
+1. Extract and prepare the ccoctl binary from the release image.
+2. [Create a ram user](https://partners-intl.aliyun.com/help/doc-detail/93720.htm)(has no permission in default) for binding the cluster components permission, and set the environment variables `CCOCTL_ALIBABA_COMPONENT_ACCESS_KEY_ID`  with the value of ram user's accesskey id, set `CCOCTL_ALIBABA_COMPONENT_ACCESS_KEY_SECRET`  with the value of ram user's accesskey secret .
+3. Choose an existing ram user who has the `ram:CreatePolicy` and `ram:AttachPolicyToUser` pemissions as least, and get this ram user's accesskey id/accesskey secret for attaching the specific component permission to the ram user created in step 2.
+4. Use the step 3 ram user’s accesskey id/secret to configure the Alibaba Cloud SDK client's creadential provider chain with [Envionment Creadentials](https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#1-environment-credentials) mode or through [Credentials File](https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#2-credentials-file) mode
+
+### Procedure
+
+1. Extract the list of CredentialsRequest custom resources (CRs) from the OpenShift Container Platform release image:
+
+   ```bash
+   $ oc adm release extract --credentials-requests --cloud=alibabacloud --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+   
```

Comment:
This functionality is not implemented yet in `oc` cli. We need to first complete that before merging this PR. Check the PR for ibmcloud [here](https://github.com/openshift/oc/pull/852).